misc/maven-resolver/src/main/java/org/ballerinalang/maven/MavenResolver.java

```diff
@@ -78,6 +79,13 @@ public MavenResolver(String targetLocation) {
         repositories = new ArrayList<>();
         repositories.add(new RemoteRepository.Builder(
                 "central", "default", "https://repo.maven.apache.org/maven2/").build());
+
+        String localRepoPath = System.getProperty("user.home") + File.separator + ".m2"
+                + File.separator + "repository";
+        if (new File(localRepoPath).exists()) {
+            repositories.add(new RemoteRepository.Builder(
+                    "local", "default", "file:" + localRepoPath).build());
```

Comment:
Can we join lines 86, 87 and 83, 84 unless it is intentional?

Reply:
83, 84 cannot be merged as it exceeds the limit. 
86, 87 can be merged